jest.config.js

```diff
@@ -10,4 +11,5 @@ module.exports = {
         },
     },
     testMatch: ['**/__tests__/**/*.spec.+(ts|tsx|js)'],
+    collectCoverage: true,
```

Comment:
this doesn't feel like a part of this PR

Reply:
indeed